src/less/chat.less

```diff
@@ -10,7 +10,14 @@
   font-size: 13px;
   position: relative;
 }
-
+#chat-log .message.whisper {
+  background: #e8e8ef;
+  border: 5px solid #545469;
```

Comment:
I'm not sold on the thicker border here. I realize it ups the visibility of the chat card quite a bit, but it adds inconsistency with the overall design of the system.

Same critique with the chosen color here as above. It just blends into the system color scheme a little to much. This needs to be a color that is gonna stand out against the systems color's (without being overly overt, either).

Reply:
I think https://colour-palette-generator.azurewebsites.net/?r=112&g=150&b=191 might work for the border here; it's a complementary color to the header background color (basically just a darker version).